### DISCUSSION TITLE
Minecraft

### DISCUSSION THREAD

**Discussion by yeyeyeyeyyeyeyeyey** [community/community#38809] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hey, is there anyone out there who can help me? I need help with getting a Minecraft file working and downloadable. Some kids have it blocked while the mean kids don't. They will not share it and keep bragging. Help plz :)

**Comment by samus-aran** [comment]
Hi @yeyeyeyeyyeyeyeyey ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on [Minecraft](https://www.minecraft.net/en-us/download). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
